ui/job-view/details/PinBoard.jsx

```diff
@@ -89,6 +112,13 @@ class PinBoard extends React.Component {
     }
     if (errorFree) {
       const jobs = Object.values(pinnedJobs);
+      // accumulate telemetry on failure classification type
+      // submit data when all jobs are counted
+      jobs.forEach((job) =>
+        gleanMetrics(failureClassificationId, job.newFailure),
+      );
+      classification.submit();
```

Comment:
Can you expand a bit on what your intent is? Is there any chance for the user to close the page before this gets hit?
Another possibility would be to submit an event for every action of interest to you.

Reply:
the intent is when a transaction takes place to record it- I am doing the submit() at the end of the transaction so if the page is closed shortly thereafter we don't lose any metrics that might be submitted on a timer.

I need to learn more about what an event is vs the counter metric I am using.

Reply:
I mean, that's fine: if you tested this and generates the data you're looking for, it's ok!